cmake/std/atdm/cee-rhel6/all_supported_builds.sh

```diff
@@ -3,8 +3,8 @@
 export ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-
 
 export ATDM_CONFIG_ALL_SUPPORTED_BUILDS=(
-  #cee-rhel6_clang-9.0.1_openmpi-4.0.2_serial_static_dbg    # SPARC has installs with this build
-  cee-rhel6_clang-9.0.1_openmpi-4.0.2_serial_static_opt    # SPARC CI build
+  #cee-rhel6_clang-9.0.1_openmpi-4.0.3_serial_static_dbg    # SPARC has installs with this build
```

Comment:
Should this be uncommented?

Reply:
> Should this be uncommented?

Once PR #6814 merges, we can set `Trilinos_SKIP_CTEST_ADD_TEST=OFF` and avoid the testing problems described in the comment below.